.github/dependabot.yml

```diff
@@ -10,6 +10,79 @@ updates:
     directory: "/"
     # The maximum number of pull requests for version updates
     open-pull-requests-limit: 10
+    groups:
+      radix-ui:
+        patterns:
+          - "@radix-ui/*"
+      graphql-codegen:
+        patterns:
+          - "@graphql-codegen/*"
+      typescript:
+        patterns:
+          - "typescript"
+          - "tsx"
+      react:
+        patterns:
+          - "react"
+          - "@types/react"
+      react-dom:
+        patterns:
+          - "react-dom"
+          - "@types/react-dom"
```

Comment:
Please put these in one `react` group.

As I see it you usually use the same version of `react` and `react-dom`.